src/styles/normalize.scss

```diff
@@ -0,0 +1,7 @@
+* {
+  box-sizing: border-box;
+}
+
+.body {
+  margin: 20px 0 0;
```

Comment:
In most cases it is impractical setting margins on `body` 
It is better to reset them and use margins/paddings with inner containers